packages/lib/src/subscriptions.ts

```diff
@@ -365,7 +365,7 @@ export class EventSubscriber<T, V=T> {
 		));
 
 		if (response.eventReplay) {
-			await this._handle(response.eventReplay);
+			await this._handle(response.eventReplay as any);
```

Comment:
This is defiantly the simplest solution. In the future we might need to review the type inference of `SubscriptionRequest` and `SubscriptionResponse` to make this work without casting to `any`. (No action needed)